charts/egeria-platform/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: egeria-platform
+description: Egeria Platform for Kubernetes
```

Comment:
after reviewing, I think the name is fine, perhaps we drop the 'for Kubernetes' (I know this is used in my other charts!) as it's superfluous